python/mxnet/symbol.py

```diff
@@ -1099,3 +1099,72 @@ def hypot(left, right):
         return _numpy.hypot(left, right)
     else:
         raise TypeError('types (%s, %s) not supported' % (str(type(left)), str(type(right))))
+
+
+def arange(start=None, stop=None, step=None, repeat=1, name=None):
+    """Simlar function in the MXNet ndarray as numpy.arange
+        See Also https://docs.scipy.org/doc/numpy/reference/generated/numpy.arange.html.
+
+    Parameters
+    ----------
+    start : number, optional
+        Start of interval. The interval includes this value. The default start value is 0.
+    stop : number, optional
+        End of interval. The interval does not include this value.
+    step : number, optional
+        Spacing between values
+    repeat : number, optional
+        "The repeating time of all elements.
+        E.g repeat=3, the element a will be repeated three times --> a, a, a.
+    ctx : Context, optional
+        The context of the NDArray, default to current default context.
+    dtype : type, optional
+        The value type of the NDArray, default to np.float32
+    Returns
+    -------
+    out : Symbol
+        The created Symbol
+    """
+    if stop is None:
+        if start is None:
+            raise ValueError("Required argument \"start\"")
+        stop = start
+        start = 0
+    if step is None:
+        step = 1
+    if start is None:
+        start = 0
+    return _internal._arange(start=start, stop=stop, step=step, repeat=repeat, name=name)
+
+
+def topk(src, axis=None, k=1, ret_typ="indices", is_ascend=False, name=None):
```

Comment:
When the `ret_typ` is set to "value", the num_outputs of topk will be 2 while the number of visible outputs will be 1. However, when I try to directly call the NDArray topk, it will still return 2 arrays instead of one array. So I've written a wrapper to deal with this special case.